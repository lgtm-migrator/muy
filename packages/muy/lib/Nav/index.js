import React from "react"
import Box from "@material-ui/core/Box"

const Nav = (props) => <Box component={"nav"} role={"navigation"} {...props} />

export default Nav
