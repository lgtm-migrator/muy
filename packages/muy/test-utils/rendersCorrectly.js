import renderer from "react-test-renderer"

const rendersCorrectly = component => {
  const tree = renderer.create(component).toJSON()
  return expect(tree).toMatchSnapshot()
}

export default rendersCorrectly
