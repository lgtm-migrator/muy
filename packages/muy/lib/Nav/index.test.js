import React from "react"
import rendersCorrectly from "../../test-utils/rendersCorrectly"
import Nav from "."

describe("<Nav/> component", () => {
  it("should render correctly with no props", () => {
    rendersCorrectly(<Nav />)
  })
})
