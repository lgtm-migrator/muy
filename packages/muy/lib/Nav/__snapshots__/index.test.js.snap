// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Nav/> component should render correctly with no props 1`] = `
<nav
  className="MuiBox-root MuiBox-root-2"
  role="navigation"
/>
`;
